src/xo-app/xosan/index.js

```diff
@@ -71,7 +71,19 @@ const XOSAN_COLUMNS = [
   },
   {
     name: _('xosanPool'),
-    itemRenderer: sr => sr.pool == null ? null : <Link to={`/pools/${sr.pool.id}`}>{sr.pool.name_label}</Link>,
+    itemRenderer: sr => sr.pool == null
+      ? null
+      : <span>
+        <Link to={`/pools/${sr.pool.id}`}>{sr.pool.name_label}</Link>
+        {sr.pool.HA_enabled && <span>
+          &nbsp;&nbsp;
+          <Tooltip content={_('highAvailability')}>
+            <span className='fa-stack'>
+              <Icon icon='pool' /><Icon icon='success' className='fa-stack-1x' />
+            </span>
+          </Tooltip>
+        </span>}
+      </span>,
```

Comment:
What's the point of this change?

Is this useful for users?

Reply:
they have to stop the HA before trying to delete the XOSAN SR, the system is in a special mode and won't let them do it.

Reply:
Implementing this logic in xo-web is a tad complicated, we have decided not to disable the button and let xo-server handle this for now.

Reply:
This change is therefore no longer necessary, can you remove it @pdonias?